app/routes/gw2.item-data.$itemId.tsx

```
@@ -0,0 +1,1093 @@
+  }
+
+  const shouldShowButton = isMobile && !chartsLoaded
+  const shouldLoadCharts = !isMobile || chartsLoaded
+
+  if (shouldShowButton) {
+    if (timeData.length > 0) {
+      return (
+        <div className="text-center my-8">
+          <button
+            onClick={loadCharts}
+            className="bg-blue-500 hover:bg-blue-700 text-white font-bold py-2 px-4 rounded">
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)